hoomd/md/PotentialMeshBond.h

```diff
@@ -0,0 +1,360 @@
+#include "hoomd/ForceCompute.h"
+#include "hoomd/GPUArray.h"
+#include "hoomd/MeshDefinition.h"
+#include <memory>
+
+#include <vector>
+
+/*! \file PotentialMeshBond.h
+    \brief Declares PotentialMeshBond
+*/
+
+#ifdef __HIPCC__
+#error This header cannot be compiled by nvcc
+#endif
+
+#include <pybind11/pybind11.h>
+
+#ifndef __POTENTIALMESHBOND_H__
+#define __POTENTIALMESHBOND_H__
+
+namespace hoomd
+    {
+namespace md
+    {
+/*! Mesh bond potential with evaluator support
+
+    \ingroup computes
+*/
+template<class evaluator> class PotentialMeshBond : public ForceCompute
```

Comment:
Aside from the fact that this class operations on "mesh bonds" instead of "bonds", are there any other differences with `PotentialBond`?